install/auto_installer.sh

```diff
@@ -0,0 +1,150 @@
+
+sudo chown -R $(whoami):$(whoami) /var/www/
+cd /var/www/
+
+# Install git and clone the repo.
+
+sudo apt-get install -y git
+git clone https://github.com/Nakiami/mellivora.git
+
+# Fetch required dependencies using Composer
+
+cd /var/www/mellivora/
+composer install
+
+# Copy and edit configuration file.
+
+cp /var/www/mellivora/include/config/config.inc.php.example /var/www/mellivora/include/config/config.inc.php
+cp /var/www/mellivora/include/config/db.inc.php.example /var/www/mellivora/include/config/db.inc.php
+
+echo "======================================================"
+read -n 1 -p "Would you like to edit mellivora config.inc.php file? (y/n) " ans;
+echo ""
+case $ans in
+    y|Y)
+        sudo nano /var/www/mellivora/include/config/config.inc.php;;
+    n|N)
+	echo ""
+	echo ""
+	echo "Ok, if you wish to modify it later the location is:"
+	echo "/var/www/mellivora/include/config/config.inc.php"
+	read -n 1 -s -r -p "Press any key to continue"
+        ;;
+    *)
+        exit;;
+esac
+
+# Make the writable directory writable.
+
+sudo chown -R www-data:www-data /var/www/mellivora/writable/
+
+# Copy and edit the Apache2 config file.
+
+sudo cp /var/www/mellivora/install/mellivora.apache.conf /etc/apache2/sites-available/mellivora.conf
+
+read -n 1 -p "Would you like to edit Apache2 config file? (y/n) " ans;
+
+case $ans in
+    y|Y)
+        sudo nano /etc/apache2/sites-available/mellivora.conf;;
+    n|N)
+	echo ""
+	echo ""
+	echo "Ok, if you wish to modify it later the location is:"
+	echo "/etc/apache2/sites-available/mellivora.conf"
+	read -n 1 -s -r -p "Press any key to continue"
+        ;;
+    *)
+        exit;;
+esac
+
+
+# Disable the default Apache site. Activate Mellivora. Restart Apache.
+
+sudo a2dissite 000-default
+sudo a2ensite mellivora
+sudo service apache2 restart
+
+
+# Create the Mellivora database and import the provided structure.
+
+echo "CREATE DATABASE mellivora CHARACTER SET utf8 COLLATE utf8_general_ci;" | mysql -u root -p
+mysql mellivora -u root -p < /var/www/mellivora/install/mellivora.sql
+mysql mellivora -u root -p < /var/www/mellivora/install/countries.sql
+
+
+# Create a new MySQL user.
+read -n 1 -s -r -p "Press any key to continue"
+echo "GRANT ALL PRIVILEGES ON mellivora.* TO 'YourUserName'@'%' IDENTIFIED BY 'YourPassword';" | mysql -u root -p
+echo ""
+echo "=========================================="
+echo "==========Mellivora Config File==========="
+echo "=========================================="
+echo "nano will now open so that you can edit Mellivora's config file."
+echo "Fill the DB_PASSWORD field with the password you used when installing MySQL previously."
+echo "If you want to modify it again after the installation, the location is:"
+echo "/var/www/mellivora/include/config/db.inc.php"
+read -n 1 -s -r -p "Press any key to continue"
+echo ""
+echo ""
+nano /var/www/mellivora/include/config/db.inc.php
+
+echo ""
+read -n 1 -s -r -p "Press any key to continue"
```

Comment:
Do we need this prompt?

Reply:
It is indeed an unnecessary redundancy, will be removed.